application/src/main/java/org/togetherjava/tjbot/commands/help/HelpThreadActivityUpdater.java

```diff
@@ -28,8 +21,7 @@
  */
 public final class HelpThreadActivityUpdater implements Routine {
     private static final Logger logger = LoggerFactory.getLogger(HelpThreadActivityUpdater.class);
-    private static final int SCHEDULE_MINUTES = 30;
-    private static final int ACTIVITY_DETERMINE_MESSAGE_LIMIT = 11;
+    private static final int SCHEDULE_MINUTES = 1;
```

Comment:
u reduced the number here. did u check the rate limits on changing tags? previously, where it was actually renaming channels, this was subject to heavy rate limits that we must not hit, hence 30mins

Reply:
sorry i did that to test, 30 mins was too long to wait. I'll change it back

Reply:
after talking with @Tais993 we agreed on 15 minutes. can u change it to 15mins please? thanks :)

Reply:
done